README.md

```diff
@@ -5,7 +5,7 @@
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
```

Comment:
Good job implementing the requested changes 👍

As we are required to customize the README template according to our project, please let's remove the Microverse template logo 🙏. Optionally, you can replace it with your own logo if they have one 🤗.

Reply:
Hi @abdulhamiid, thanks for your comments, however as this is a Microverse project I would really like to keep that Microverse logo there as it nicely shows the organization I'm doing this project for. Please let me know if you have any other comments.

Reply:
Hi @dennis-every, you have mentioned a good point, however it is part of Microverse **terms and conditions** that every student using her README template should remove the Microverse template logo. and they can replace it with their own logo if they have one.

Thanks 🤗.

Reply:
Hi @abdulhamiid, thanks for your reply and explanation - makes sense. I have removed the Microverse logo from the readme. Thanks 🤗

Reply:
Cheers and Happy coding! 👏👏👏